internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -424,6 +427,22 @@ func (c *Coordinator) Run(ctx context.Context) error {
 // information about the state of the Elastic Agent.
 func (c *Coordinator) DiagnosticHooks() diagnostics.Hooks {
 	return diagnostics.Hooks{
+		{
+			Name:        "local-config",
+			Filename:    "local-config.yaml",
```

Comment:
Let's document this file in the official docs https://github.com/elastic/ingest-docs/blob/main/docs/en/ingest-management/commands.asciidoc#elastic-agent-diagnostics-command